site/_posts/2017-06-12-Where-have-all-the-providers-gone.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Last Week in ManageIQ: Where have all the Providers gone"
+author: durandom
+date: 2017-06-12
+comments: true
+published: true
+tags: LWIMIQ
+---
+:guitar: Before you continue reading, I want you to click [here](https://youtu.be/ZgXNVA9ngx8) to set the mood :metal:
+
+```
+Where have all the Providers gone? 
```

Comment:
:heart: I don't want to admit how often I parody this song in my head :laughing: